src/main/java/by/exadel/internship/controller/UserController.java

```diff
@@ -27,9 +25,16 @@
         return userService.getAll();
     }
 
-    @GetMapping("/{userId}")
+    @GetMapping("/userId")
     @ApiOperation("Return user by id")
-    public UserDTO getUserById(@PathVariable UUID userId) {
+    public UserDTO getUserById(@RequestParam UUID userId) {
         return userService.getById(userId);
     }
+
+    @GetMapping("/internshipId")
```

Comment:
internshipId should be query parameter, not part of the path

Reply:
done